src/NuGetForUnity/Editor/NugetHelper.cs

```diff
@@ -67,6 +67,8 @@ public static class NugetHelper
         /// </summary>
         private static readonly List<NugetPackageSource> packageSources = new List<NugetPackageSource>();
 
+        public static event Action OnInstalledPackagesChanged;
```

Comment:
I remember there being a code earlier that would automatically trigger Restore when packages.config file was changed and we thought we needed this event because of that case but it seems that code is now only triggered the first time project is loaded.

Is that right? Does that mean if I install some packages and push packages.config other developers won't automatically get that package installed when they pull my changes if they already have project open in Unity and they will have to manually run Restore?

Reply:
Yes for `packages.config` change detection we would need to implement a AssetPostprocessor / add this feature to the `NugetPackageAssetPostprocessor`